style.css

```diff
@@ -1 +1,217 @@
 /*  Add your styling here */
+
+#navigation{
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+    position: fixed;
+    align-items: center;
+    top: 0px;
+    background-color: white;
+    overflow: hidden;
+    height: 60px;
+}
+#logo{
+    margin-top: 0px;
+    padding-top: 0px;
+
+    height: 50px;
+    width: 120px;
+}
+a{
+    text-decoration:none;
+    text-decoration: none;
+    color: gray;
+}
+#first-section{
+    height: 550px;
+    width: 100%;
+    background-color:rgb(214, 235,215);
+    margin-top: 50px;
+    
+}
+#first-text{
+    margin-top: 0px;
+}
+.title{
+    font-family: 'Anonymous Pro', monospace;
+    font-family: 'PT Serif', serif;
+    font-size: 35px;
+    color: grey;
+    margin: 0px 30px 0px 30px;
+    padding: 20px 5px 5px 5px;
+    text-align: center
+            
+}
+.text{
+    font-family: 'Open Sans Condensed', sans-serif;
+    font-size: 25px;
+    color: grey;
+    margin-top: 0px;
+    padding: 5px;
+    margin: 0px 30px 0px 30px;
+    text-align: center;
+    
+    
+}
+body{
+    margin-top: 10px;
+    margin-left: 0px;
+    width: 480px;
+    
+}
+#second-section{
+    /* height: 235px; */
+    width: 100%;
+    background-color:#f5f6fa;
+    display: flex;
+    flex-direction:column;
+    /* position: relative; */
+    margin-top: 10px;
+    align-items: center;
+}
+
+.second-section-img{
```

Comment:
Continue to favor using classes where possible in order to encourage re-usability of styles like this. Applying css using id selectors is mainly used when wanting to override class styles.